packages/noports_core/test/sshnp/sshnp_test.dart

```diff
@@ -1,7 +1,43 @@
+import 'package:mocktail/mocktail.dart';
+import 'package:noports_core/sshnp_foundation.dart';
 import 'package:test/test.dart';
 
+import 'sshnp_mocks.dart';
+
+class StubbedSshnp extends Mock implements Sshnp {}
+
+class MySshnpResult extends SshnpResult {}
+
 void main() {
   group('Sshnp', () {
-    test('public API test', () {});
-  });
+    late final StubbedSshnp stubbedSshnp;
+    late final MockAtClient mockAtClient;
+    late final MockSshnpParams mockSshnpParams;
+    late final MySshnpResult mySshnpResult;
+    late final SshnpDeviceList sshnpDeviceList;
+
+    setUp(() {
+      stubbedSshnp = StubbedSshnp();
+      mockAtClient = MockAtClient();
+      mockSshnpParams = MockSshnpParams();
+      mySshnpResult = MySshnpResult();
+      sshnpDeviceList = SshnpDeviceList();
+    });
+
+    test('public API test', () async {
+      /// Just test that the public members are there,
```

Comment:
For the Sshnp interface, we only need to test that the public API members are there, since the tests for SshnpCore cover all logic.